### DISCUSSION TITLE
Issue with TinyMce in Common BlockData

### DISCUSSION THREAD

**Discussion by dangngocminh88** [Epicweb-Optimizely/Epicweb.Optimizely.AIAssistant#3] [Q&A]
I have a common BlockData with a XhtmlString to embed in multi other BlockData. When I add AI Assistant Button to that XhtmlString, I got the error "AI Assistant buttons not loaded, this object has not a IContent and therefore will not work." in console and AI Button does not show in TinyMce as expected.
How can I fix it?
![image](https://github.com/user-attachments/assets/19e1a9ce-f72f-43a6-8b20-3000c4f86d37)
![image](https://github.com/user-attachments/assets/9d087b87-5e58-45e6-80b8-725aa61dba67)
I'm using Epicweb.Optimizely.AIAssistant 1.13.6

**Comment by LucGosso** [comment]
@dangngocminh88  hi! We will need to address this. 

There is on known problem, when creating a new block (before saving the block), the context is not known in the dojo UI (it will point to startpage or the last context clicked). So prompts like "summarize this page" wont work. That is why we turn it of if the block do not has an Id. 

I quess it is not working too with inline blocks, but that we might be able to handle. 

Can you tell me more how you are using it? Is it an inline block?

**Comment by LucGosso** [comment]
This is solved in next version 1.14 coming early 2025. Though, it will not translate inline local blocks out of the box, but that will be in next Episerver.Labs.LanguageManager (ive been told)
